communication.md

```diff
@@ -11,7 +11,7 @@ The Kubernetes community abides by the [CNCF code of conduct].  Here is an excer
 ## SIGs
 
 Kubernetes encompasses many projects, organized into [SIGs].
-Helm reports to [SIG-Apps] which has its own communication
+Helm reports to Kubernetes' [SIG-Apps] which has its own communication
```

Comment:
I guess this should be changed now to the CNCF? cc @mattfarina

Reply:
@prydonius good point. But we may rephrase this completely as CNCF projects do not "report" to CNCF :)

Reply:
Should we just remove the whole SIGs section?

Reply:
@prydonius good idea.